be/src/olap/rowset/alpha_rowset.cpp

```diff
@@ -110,7 +110,7 @@ OLAPStatus AlphaRowset::link_files_to(const FilePathDesc& dir_desc, RowsetId new
     return OLAP_SUCCESS;
```

Comment:
not support alpha rowset any more, just return error and stop migration job if this rowset is alpharowset